docs/config/01-overview.md

```diff
@@ -519,4 +519,29 @@ Passes custom configuration down to rollup itself. The following options can be
 - `inputOptions`: [`context`](https://rollupjs.org/configuration-options/#context), [`external`](https://rollupjs.org/configuration-options/#external), [`moduleContext`](https://rollupjs.org/configuration-options/#modulecontext) [`treeshake`](https://rollupjs.org/configuration-options/#treeshake)
 - `outputOptions`: [`globals`](https://rollupjs.org/configuration-options/#output-globals) 
 
-*default: `{}`*
\ No newline at end of file
+*default: `{}`*
+
+## watchIgnoredRegex
+
+_default: `[]`_
+
+_type: `RegExp | RegExp[]`_
```

Comment:
Just to be aligned with previous formatting even though it will turn out to be the same
```suggestion
*default: `[]`*

*type: `RegExp | RegExp[]`*
```